_data/init-cli/docker_init.yaml

```diff
@@ -60,91 +62,104 @@ examples: |-
-  The following appears after selecting `Other`.
+  The following example shows the prompts that appear after selecting `Go` and example input.
   
   ```console
+  ? What application platform does your project use? Go
+  ? What version of Go do you want to use? 1.20
+  ? What's the relative directory (with a leading .) of your main package? .
+  ? What port does your server listen on? 3333
+
   CREATED: .dockerignore
   CREATED: Dockerfile
-  CREATED: docker-compose.yaml
+  CREATED: compose.yaml
   
   ✔ Your Docker files are ready!
   
   Take a moment to review them and tailor them to your application.
   
   When you're ready, start your application by running: docker compose up --build
-  ```
   
-  ### Example of selecting Go
+  Your application will be available at http://localhost:3333
+  ```
   
-  The following example runs `docker init` and walks through the options after selecting `Go`.
+  ### Example of selecting Python
+
+  The following example shows the prompts that appear after selecting `Python` and example input.
   
   ```console
-  $ docker init
-  
-  Welcome to the Docker Init CLI!
+  ? What application platform does your project use? Python
+  ? What version of Python do you want to use? 3.8
+  ? What port do you want your app to listen on? 8000
+  ? What is the command to run your app (e.g., gunicorn 'myapp.example:app' --bind=0.0.0.0:8000)? python ./app.py
   
-  This utility will walk you through creating the following files with sensible defaults for your project:
-    - .dockerignore
-    - Dockerfile
-    - docker-compose.yaml
+  CREATED: .dockerignore
+  CREATED: Dockerfile
+  CREATED: compose.yaml
   
-  Let's get started!
+  ✔ Your Docker files are ready!
   
-  ? What application platform does your project use?  [Use arrows to move, type to filter]
-  > Go - (detected) suitable for a Go server application
-    Other - general purpose starting point for containerizing your application
-    Don't see something you need? Let us know!
-    Quit
-  ```
+  Take a moment to review them and tailor them to your application.
   
-  The following appears after selecting `Go`.
+  When you're ready, start your application by running: docker compose up --build
   
-  ```console
-  ? What application platform does your project use? Go
-  ? What version of Go do you want to use? (1.20)
+  Your application will be available at http://localhost:8000
   ```
   
-  The following appears after selecting the default `1.20`.
+  ### Example of selecting Node
+  
+  The following example shows the prompts that appear after selecting `Node` and example input.
   
   ```console
-  ? What version of Go do you want to use? 1.20
-  ? What's the relative directory (with a leading .) of your main package? (.)
-  ```
+  ? What application platform does your project use? Node
+  ? What version of Node do you want to use? 18
+  ? Which package manager do you want to use? yarn
+  ? What command do you want to use to start the app? node index.js
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Is this the latest node prompt? There _should_ be another step asking if they want to build the app first.